### PR TITLE
Abrar/mysqlconnector/affirm 3.3.2

### PR DESCRIPTION
**Pull request by abrarsher23**

### CODE REVIEW

**Review by nickwu241 — COMMENTED** [review]
How much larger did the package become?

File: pom.xml [hunk 1]
```diff
@@ -437,6 +437,11 @@
       <artifactId>tabular-client-runtime</artifactId>
       <version>1.18.4</version>
     </dependency>
+    <dependency>
```
Comment (abrarsher23) [inline pom.xml:440]: will attempt removing this

Comment (rayhondo) [reply pom.xml:440]: definitely need this!

Comment (rayhondo) [reply pom.xml:440]: just tested without it and it did not work

**Pull request merged**